cypher-shell/src/test/java/org/neo4j/shell/cli/InteractiveShellRunnerTest.java

```diff
@@ -66,6 +69,9 @@ public void setup() throws Exception {
         txHandler = mock(TransactionHandler.class);
         historyFile = temp.newFile();
         badLineError = new ClientException("Found a bad line");
+        userMessagesHandler = mock(UserMessagesHandler.class);
+        when(userMessagesHandler.getWelcomeMessage()).thenReturn("Welcome to cypher-shell!");
```

Comment:
Could you add a test in this file which verifies that the shell prints the exit message as well?